geemap/toolbar.py

```diff
@@ -27,211 +31,130 @@
 
 
 @map_widgets.Theme.apply
-class Toolbar(widgets.VBox):
-    """A toolbar that can be added to the map."""
-
-    @dataclass
-    class Item:
-        """A representation of an item in the toolbar.
+class ToolbarItem(anywidget.AnyWidget):
+    """A toolbar item widget for geemap."""
+
+    _esm = pathlib.Path(__file__).parent / "static" / "toolbar_item.js"
+    active = traitlets.Bool(False).tag(sync=True)
+    icon = traitlets.Unicode("").tag(sync=True)
+    tooltip_ = traitlets.Unicode("").tag(sync=True)
+
+    def __init__(
+        self,
+        icon: str,
+        tooltip: str,
+        callback: Callable[[Any, bool, Any], None],
+        control: Optional[widgets.Widget] = None,
+        reset=False,
+        active=False,
+    ):
+        """A togglable, toolbar item.
 
-        Attributes:
-            icon: The icon to use for the item, from https://fontawesome.com/icons.
+        Args:
+            icon (str): The icon hexcode to use, from https://fonts.google.com/icons.
```

Comment:
The `icon` parameter can be the point/hex code or icon name?

Reply:
Oops, stale documentation! Done.